### PR TITLE
updated cli bullet

### PR DESCRIPTION
**Pull request by mirnawong1**
added link to cli and spelled it out further for clarity

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`
- [ ] [Run link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/reference/dbt-commands.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: "dbt Command reference"
 
 dbt is typically run one of two ways:
 * In [dbt Cloud](/docs/get-started/develop-in-the-cloud)
-* On the command line
+* On the [Command Line Interface](/docs/get-started/about-the-cli) (CLI)
```
Comment (nghi-ly) [inline website/docs/reference/dbt-commands.md]:
```suggestion
* On the [command line interface](/docs/get-started/about-the-cli) (CLI)
```

Comment (nghi-ly) [reply website/docs/reference/dbt-commands.md]: similar to GUI, which stands for graphical user interface

**Review by nghi-ly — APPROVED** [review]

File: website/docs/reference/dbt-commands.md (hunk 1 above)
Comment (mirnawong1) [reply website/docs/reference/dbt-commands.md]: ah ok, i originally had it as lowercased but then i double checked and went with the styling on `[this page](https://docs.getdbt.com/docs/get-started/about-the-cli)` 🙈

would it make sense to open a pr to edit that page above and correct?

Comment (nghi-ly) [reply website/docs/reference/dbt-commands.md]: good flag, @mirnawong1 ! i opened PR #2482 for it

**Pull request merged**